package.xml

```diff
@@ -13,6 +13,7 @@
   <author>Aaron Blasdel</author>
   <author>Dirk Thomas</author>
   <author>Thibault Kruse</author>
+  <author email="ivanpauno@ekumenlabs.com">Ivan Paunovic</author>
```

Comment:
I prefer not being listed as author, as I'm really not.
I know that the current policy is to add previous maintainers as authors, but honestly I think I merged two (?) PRs during that period.

Reply:
OK, I've removed you in ca97aea.

Reply:
thanks